uiowa/uiowa
# i3011 transfer tips

Closes #3011 


# How to test


1. blt ds --site=admissions.uiowa.edu
2. blt frontend
3. Add a [transfer tip](http://admissions.local.drupal.uiowa.edu/node/add/transfer_tips)
4. Go to an area of study page (http://admissions.local.drupal.uiowa.edu/node/956/edit) and search for the transfer tip you added previously. Add it, save to page.
5. See that the 'transfer tip' link displays in the 'Transfer' card under the 'Admission Process' section of AOS page
6. Click on it -> make sure it takes you to full view of transfer tip ct
7. Make sure the AOS button is displayed and when you click on it, it takes you back to the full aos page
8. Masquerade as webmaster
8. Add a new page
9. Go to layout -> add a 'more' block and search for the transfer tips view
10. Add transfer tips view block to page and save the page
11. Make sure the transfer tips you've added display as an accordion on the page
